frontend/src/types.ts

```diff
@@ -1387,6 +1387,7 @@ export interface ExperimentResults {
     probability: Record<string, number>
     filters: FilterType
     itemID: string
+    noData?: boolean
```

Comment:
where is this set? I can't find it anywhere O.o

Reply:
it should be in the logic file under "no_data" error :o

Reply:
share screen after planning? O.o

Reply:
just kidding, that was before I removed it to set no data error results to "null" instead lol. I will remove this in my trends PR 😁